contracts/LockedFarming.sol

```diff
@@ -274,9 +286,34 @@ contract SMD_v5 is Ownable {
             _lockDuration,
             _rewardAmount
         );
+
+        isPaused = false;
```

Comment:
Is this necessary? Will there ever be a case where we set a new period, but we don't want to unpause the contract?

Reply:
yes required cause[ in constructor](https://github.com/Seedifyfund/Locked-Yield-Farming/blob/838c5149cafc3786258ded7d763968fa04b0e020/contracts/LockedFarming.sol#L185) is set to `true`

to my understanding there will not be a case where the new period is set but we do not want to run it straight away - in fact before that this variables was always put to `false` but in a sub method so I have not change the base logic at all (only refactored)

Reply:
Nice one, that's all then. Review passed. 👍